### ISSUE TITLE
[QUESTION] JOINED inheritance type

### ISSUE THREAD

**Issue by versh23** [DamienHarper/DoctrineAuditBundle#114]
Hello!

i have two entity with `@ORM\InheritanceType("JOINED")`

```
/**
 * @ORM\Entity()
 * @Auditable()
 */
class Child extends Base

```

```
/**
 * @ORM\Entity()
 * @ORM\InheritanceType("JOINED")
 * @ORM\DiscriminatorColumn(name="kind", type="string")
 * @ORM\DiscriminatorMap({"child" = "App\Entity\Child"})
 */
abstract class Base
```

When i add `@Auditable `annotation, bundle want to create two tables. 
**child_audit** and **base_audit**
Why ? 
is it necessary for what? 
There is also descriminator column and its always null if  inheritance type === joined

Can you check is it bug or not ?

**Comment by versh23** [comment]
i think https://github.com/DamienHarper/DoctrineAuditBundle/blob/master/src/DoctrineAuditBundle/Event/CreateSchemaListener.php#L55 is not nessesary

**Comment by DamienHarper** [comment]
@versh23 Class Table Inheritance (Joined inheritance) in not supported as of now. The bundle currently only supports Mapped Superclass and Single Table inheritance.
Though, I'd be happy to review such a PR.

**Comment by DamienHarper** [comment]
Fixed by #115
